lib/httpd_dbus_api.rb

```diff
@@ -1,28 +1,28 @@
 require "faraday"
 require "json"
 
-class HttpdAuthApi
+class HttpdDBusApi
   def initialize(options = {})
     @options = options
   end
 
   def user_attrs(userid, attributes = nil)
     if attributes.present?
-      auth_api("/api/user_attrs/#{userid}?attributes=#{attributes.join(',')}")
+      dbus_api("/api/user_attrs/#{userid}?attributes=#{attributes.join(',')}")
```

Comment:
I think it's better to use URI methods to construct this than doing it manually, in order to ensure that things are escaped properly, but that can come later